tests/test_logo.py

```diff
@@ -0,0 +1,10 @@
+from io import BytesIO
+
+
+def test_logo(mocker, app, client, auth_header):
```

Comment:
can we get tests for:

* request returns 404
* file is returned but it isn't a valid svg?